### PR TITLE
Integrate with coverage

### PR DESCRIPTION
**Pull request by masayukig**
This commit adds `--coverage` option to integrate with `coverage`.

Fixes Issue #97

### CODE REVIEW

File: stestr/commands/run.py [hunk 1]
```diff
@@ -318,19 +334,32 @@ def run_command(config='.stestr.conf', repo_type='file',
...
+                               subunit_out=subunit_out,
+                               repo_type=repo_type,
+                               repo_url=repo_url, run_id=combine_id,
+                               pretty_out=pretty_out,
+                               color=color, stdout=stdout,
+                               abbreviate=abbreviate,
+                               suppress_attachments=suppress_attachments)
+            if coverage:
+                os.system("coverage combine")
+                os.system("coverage html -d ./cover %s" % omit)
```
Comment (mtreinish) [inline stestr/commands/run.py]: I don't think that we want to generate html output like this from a CLI tool. If we're going to generate the report (and not just collect coverage)  I think we should call coverage report to print to stdout instead.

File: stestr/commands/run.py [hunk 2]
```diff
@@ -318,19 +334,32 @@ def run_command(config='.stestr.conf', repo_type='file',
...
+            result = load.load(in_streams=run_proc,
+                               subunit_out=subunit_out,
+                               repo_type=repo_type,
+                               repo_url=repo_url, run_id=combine_id,
+                               pretty_out=pretty_out,
+                               color=color, stdout=stdout,
+                               abbreviate=abbreviate,
+                               suppress_attachments=suppress_attachments)
+            if coverage:
+                os.system("coverage combine")
```
Comment (mtreinish) [inline stestr/commands/run.py]: Is there a python interface/api we could call instead of shelling out?

**Comment by masayukig** [comment]
> That being said I have been thinking about this some more and wonder if this goes against the unix philosophy for stestr. stestr is a parallel python test runner and it should do that well, while coverage collection and reporting is the role of the coverage tool.

Yeah, good point.
After reading your comment, I think we don't need this function so far because this PR brings us little benefit.

I was thinking this function could be useful a bit, especially for original `.... testr --coverage` users. But, now, I think the benefit is not that much. And, yea, we can do same things with the coverage tool. Another concern is that this could introduce another complexity to stestr itself..

So, let's close this PR and the issue #97 ?

**Comment by mtreinish** [comment]
Well, I'm not sure. At one point (mainly back when #97 was opened) I felt like it was in scope for stestr. But, now if feels out of scope to me. So I guess we should close it and if we change our minds again in the future :) we can always reopen it.

**Comment by masayukig** [comment]
yeah, agree. Let's close both of the PR and issue at this moment.

**Pull request closed without merging**